libs/spark/src/Unstable_SectionMessage/Unstable_SectionMessage.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import type { Meta, Story } from '@storybook/react/types-6-0';
+import { Unstable_SectionMessage } from '..';
+
+export default {
+  title: '@ps/Unstable_SectionMessage',
+  component: Unstable_SectionMessage,
+} as Meta;
+
+const Template = (args) => (
+  <Unstable_SectionMessage {...args}>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+  </Unstable_SectionMessage>
+);
+
+export const Configurable: Story = Template.bind({});
+
+export const Default: Story = Template.bind({});
+Default.storyName = '(default)';
+
+export const SeverityInfo: Story = Template.bind({});
+SeverityInfo.args = { severity: 'info' };
+SeverityInfo.storyName = 'severity=info';
+const onClose = () => {
```

Comment:
```suggestion
SeverityInfo.storyName = 'severity=info';

const onClose = () => {
```
formatting preference